docs/sources/features/panels/logs.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Logs Panel"
+keywords = ["grafana", "dashboard", "documentation", "panels", "logs panel"]
+type = "docs"
+aliases = ["/reference/logs/"]
+[menu.docs]
+name = "Logs"
+parent = "panels"
+weight = 2
++++
+
+
+# Logs Panel
+
+<img class="screenshot" src="/assets/img/features/logs-panel.png">
+
+> Logs panel is only available in Grafana v6.4+
+
+The logs panel shows log lines from datasources that support logs, e.g., Elastic, Influx, and Loki.
+Typically you would use this panel next to a graph panel to display the log output of a related process.
+
+## Querying Data
+
+The logs panel will show the result of queries that are specified in the **Queries** tab.
+The results of multiple queries will be merged and sorted by time.
+Note that you can scroll inside the panel in case the datasource returns more lines than can be displayed at any one time.
+
+### Query Options
+
+Some datasources (e.g., Loki) allow the use of **Live** tailing to show a steady stream of log messages.
+Note that this can put extra effort on the datasource and your browser.
```

Comment:
Maybe it would also make sense to add that with live tailing the dashboard time range will be ignored.